Source/Parsing/Frame/Content/Operation/ID3CommentFrameContentParsingOperation.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ID3CommentFrameContentParsingOperation.swift
+//  ID3TagEditor
+//
+//  Created by Nolaine Crusher on 3/7/20.
+//  Copyright © 2020 Fabrizio Duroni. All rights reserved.
+//
+
+import Foundation
+
+struct ID3CommentFrameContentParsingOperation: FrameContentParsingOperation {
+
+  let frameName: FrameName
+
+  func parse(frame: Data, version: ID3Version, completed: (FrameName, ID3Frame) -> ()) {
+    var parsing = frame[...]
+    let name = parsing.extractFirst(version.identifierSize)
+    assert(
+      String(ascii: name) == frameName.fourByteString
+        || String(ascii: name) == frameName.threeByteString,
+      "Mismatched frame name: \(String(ascii: name)) ≠ \(frameName.fourByteString!)/\(frameName.fourByteString!)"
+    )
+    _ = parsing.extractFirst(version.identifierSize) // Size
+
+    #warning("Should be able to throw or otherwise signal that the encoding flag was invalid. Falling back to the most standard encoding could cause trouble down the line.")
```

Comment:
Okay, this is a little close to what's over my head. If the assertion fails, then add the error, or throw?

Reply:
What to do about corruptions like this would be an upstream decision. (It would mean changing the method signatures of widely‐used protocols.)